### ISSUE TITLE
Scale the line length?

### ISSUE THREAD

**Issue by stodge** [w8r/leaflet-labeled-circle#24]
I'm not sure if I can explain this very well, but it would be interesting to have an option that scales the length of the line (line between marker and label circle). So when you zoom in and out the line length stays the same relative to the screen, not the map. I'll go away and see if I can think how to reword this as I'm not explaining it very well.

**Comment by w8r** [comment]
It would be hard to find the annotation then, it would mean that it has it's own geo-reference, and it's not the point, it's a part of UI, if you will. You sure you need that and not just a polyline and another marker?
